azure-spring-boot-samples/spring-data-gremlin-boot-sample/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>azure-spring-boot-samples</artifactId>
+        <groupId>com.microsoft.azure</groupId>
+        <version>0.0.1-SNAPSHOT</version>
```

Comment:
Version should be defined in one place, then refer it by property

Reply:
We would like to refine it for all samples in another PR.